src/transport_performance/population/rasterpop.py

```diff
@@ -16,6 +16,12 @@
 from shapely.geometry.polygon import Polygon
 from matplotlib import colormaps
 from cartopy.mpl.geoaxes import GeoAxes
+from transport_performance.utils.defence import (
+    _is_expected_filetype,
+    _type_defence,
+    _handle_path_like,
+    _check_parent_dir_exists,
+)
 
 
 class RasterPop:
```

Comment:
### Suggested Change  
_NOTE: This change is completely optional and doesn't fix any broken functionality_  

To improve usability, the `which` parameter could be lowered (`.lower()`) once it is passed to `.plot()`. This would disregard cases where the user uses the wrong capitalisation.  

This is an overall QOL improvement, but I'm happy for it to not be implemented if you feel that it is unnecessary.

Reply:
Great shout! I'll add this in.

Reply:
This has been implemented in commit 3f8dfac. There is a corresponding unit test too. Let me know if you consider this conversation closed.

Reply:
Resolving this conversation as it's complete from commit `3f8dfac`